store.go

```diff
@@ -1508,9 +1508,8 @@ func (s *store) Version() ([][2]string, error) {
 }
 
 func (s *store) Mount(id, mountLabel string) (string, error) {
-	rcstore, err := s.ContainerStore()
-	if err != nil {
-		return "", err
+	if layerID, err := s.ContainerLayerID(id); err == nil {
+		id = layerID
 	}
```

Comment:
I believe this matches the current code.  Where we only replaced the id field if it was a container, otherwise we just used it as a layer.